pkg/driverbuilder/local.go

```diff
@@ -181,59 +190,58 @@ func (lbp *LocalBuildProcessor) Start(b *builder.Build) error {
 			cmd.Env = append(cmd.Env, fmt.Sprintf("%s=%s", key, val))
 		}
 
-		stdout, err := cmd.StdoutPipe()
-		if err != nil {
-			lbp.Logger.Warn("Failed to pipe stdout", lbp.Logger.Args("err", err))
-			_, err = cmd.CombinedOutput()
-		} else {
-			cmd.Stderr = cmd.Stdout // redirect stderr to stdout so that we catch it
-			defer stdout.Close()
-			err = cmd.Start()
-			if err != nil {
-				lbp.Logger.Warn("Failed to execute command", lbp.Logger.Args("err", err))
-			} else {
-				// print the output of the subprocess line by line
-				scanner := bufio.NewScanner(stdout)
-				for scanner.Scan() {
-					m := scanner.Text()
-					fmt.Println(m)
-				}
-				err = cmd.Wait()
-			}
+		out, err := cmd.CombinedOutput()
```

Comment:
No need to pipe and scan stdout/stderr: we are always behind a stylish spinner now, therefore we will never print these lines in real time.